### DISCUSSION TITLE
Paid Professional Support & Tutorial on Live Chat, Please

### DISCUSSION THREAD

**Discussion by tchannon55** [LiveHelperChat/livehelperchat#1977] [General]
Hello. I have taken over a web site for a client and the client uses LHC software.  It is not working and neither myself nor my developers are able to figure out the roadblocks on why it is not working smoothly.

Please see https://www.loom.com/share/06a3e540052d459981519f61db512add (I recorded this for our developers). 

If someone is able to give me a walk through and/or help troubleshoot, I will gladly compensate by the hour for assistance.

The biggest problem is that users are logging in as operators and they are unable to take chats from customers.

**Comment by remdex** [comment]
You are missing some permissions for the chat. I can give support just skype me, my nick remdex. Or you can just contact via discord

**Reply by tchannon55** [reply]
Can't find you on skype. How do I contact support?

**Reply by tchannon55** [reply]
I found you, I think and sent you a Skype message.

**Reply by tchannon55** [reply]
Ok I am on Skype and in Discord
